packages/mdc-tabs/tab-bar/mdc-tab-bar.scss

```diff
@@ -78,34 +78,3 @@
 .mdc-tab-bar--icons-with-text {
   height: 72px;
 }
-
-.mdc-tab-bar--indicator-primary,
```

Comment:
removing these --primary and --accent is a breaking change, so remember to add that to the commit